omaha_server/crash/senders.py

```diff
@@ -41,15 +41,26 @@ class ELKSender(BaseSender):
     name="ELK"
     handler = None
 
-    def send(self, message, extra={}, tags={}, data={}, crash_obj=None):
-        logger = logging.getLogger('crashes')
-        extra.update(tags)
-        # sentry.interfaces.Exception
-        extra.update(data)
-        extra.update({
-            'signature': message
-        })
-        logger.info(add_extra_to_log_message(message, extra=extra))
+    def send(self, message, extra={}, tags={}, sentry_data={}, crash_obj=None):
+            logger = logging.getLogger('crashes')
+
+            extra.update(tags)
+            extra['sparrow_version'] = tags['ver'] if 'ver' in tags else 'unknown'
+
+            # We don't want "sentry.interfaces" or other sentry specific things as part of any field name.
+            extra['exception'] = str(sentry_data.get('sentry.interfaces.Exception'))
+            extra['user'] = sentry_data.get('sentry.interfaces.User') # will be 'None' if no user in sentry_data.
+            # User is passed in as "dict(id=crash.userid)". Unpack.
+            if type(extra['user']) is dict: 
+                extra['user'] = extra['user'].get('id')
+
+            # The "message" is actually a crash signature, not appropriate for the ELK "message" field.
+            extra['signature'] = message
+            # All ELK messages are expected to include logger_name.
+            extra['logger_name'] = 'omaha_server'
+
+            # Send message with logger.
+            logger.info(add_extra_to_log_message("Sparrow Crashes", extra=extra))
```

Comment:
Please use an environment variable to set app specific crash message(e.g. Sparrow crashes).